src/env_var.zig

```diff
@@ -0,0 +1,656 @@
+pub const BUN_CONFIG_DNS_TIME_TO_LIVE_SECONDS = New(kind.unsigned, "BUN_CONFIG_DNS_TIME_TO_LIVE_SECONDS", .{ .default = 30 });
+pub const BUN_CRASH_REPORT_URL = New(kind.string, "BUN_CRASH_REPORT_URL", .{});
+pub const BUN_DEBUG = New(kind.string, "BUN_DEBUG", .{});
+pub const BUN_DEBUG_ALL = New(kind.boolean, "BUN_DEBUG_ALL", .{});
+pub const BUN_DEBUG_CSS_ORDER = New(kind.boolean, "BUN_DEBUG_CSS_ORDER", .{ .default = false });
+pub const BUN_DEBUG_ENABLE_RESTORE_FROM_TRANSPILER_CACHE = New(kind.boolean, "BUN_DEBUG_ENABLE_RESTORE_FROM_TRANSPILER_CACHE", .{ .default = false });
+pub const BUN_DEBUG_HASH_RANDOM_SEED = New(kind.unsigned, "BUN_DEBUG_HASH_RANDOM_SEED", .{ .deser = .{ .error_handling = .not_set } });
+pub const BUN_DEBUG_QUIET_LOGS = New(kind.boolean, "BUN_DEBUG_QUIET_LOGS", .{});
+pub const BUN_DEBUG_TEST_TEXT_LOCKFILE = New(kind.boolean, "BUN_DEBUG_TEST_TEXT_LOCKFILE", .{ .default = false });
+pub const BUN_DEV_SERVER_TEST_RUNNER = New(kind.string, "BUN_DEV_SERVER_TEST_RUNNER", .{});
+pub const BUN_ENABLE_CRASH_REPORTING = New(kind.boolean, "BUN_ENABLE_CRASH_REPORTING", .{});
+pub const BUN_FEATURE_FLAG_DUMP_CODE = New(kind.string, "BUN_FEATURE_FLAG_DUMP_CODE", .{});
+/// TODO(markovejnovic): It's unclear why the default here is 100_000, but this was legacy behavior
+/// so we'll keep it for now.
+pub const BUN_INOTIFY_COALESCE_INTERVAL = New(kind.unsigned, "BUN_INOTIFY_COALESCE_INTERVAL", .{ .default = 100_000 });
+pub const BUN_INSPECT = New(kind.string, "BUN_INSPECT", .{ .default = "" });
+pub const BUN_INSPECT_CONNECT_TO = New(kind.string, "BUN_INSPECT_CONNECT_TO", .{ .default = "" });
+pub const BUN_INSPECT_PRELOAD = New(kind.string, "BUN_INSPECT_PRELOAD", .{});
+pub const BUN_INSTALL = New(kind.string, "BUN_INSTALL", .{});
+pub const BUN_INSTALL_BIN = New(kind.string, "BUN_INSTALL_BIN", .{});
+pub const BUN_INSTALL_GLOBAL_DIR = New(kind.string, "BUN_INSTALL_GLOBAL_DIR", .{});
+pub const BUN_NEEDS_PROC_SELF_WORKAROUND = New(kind.boolean, "BUN_NEEDS_PROC_SELF_WORKAROUND", .{ .default = false });
+pub const BUN_OPTIONS = New(kind.string, "BUN_OPTIONS", .{});
+pub const BUN_POSTGRES_SOCKET_MONITOR = New(kind.string, "BUN_POSTGRES_SOCKET_MONITOR", .{});
+pub const BUN_POSTGRES_SOCKET_MONITOR_READER = New(kind.string, "BUN_POSTGRES_SOCKET_MONITOR_READER", .{});
+pub const BUN_RUNTIME_TRANSPILER_CACHE_PATH = New(kind.string, "BUN_RUNTIME_TRANSPILER_CACHE_PATH", .{});
+pub const BUN_SSG_DISABLE_STATIC_ROUTE_VISITOR = New(kind.boolean, "BUN_SSG_DISABLE_STATIC_ROUTE_VISITOR", .{ .default = false });
+pub const BUN_TCC_OPTIONS = New(kind.string, "BUN_TCC_OPTIONS", .{});
+pub const BUN_TMPDIR = New(kind.string, "BUN_TMPDIR", .{});
+pub const BUN_TRACK_LAST_FN_NAME = New(kind.boolean, "BUN_TRACK_LAST_FN_NAME", .{ .default = false });
+pub const BUN_TRACY_PATH = New(kind.string, "BUN_TRACY_PATH", .{});
+pub const BUN_WATCHER_TRACE = New(kind.string, "BUN_WATCHER_TRACE", .{});
+pub const CI = New(kind.string, "CI", .{});
```

Comment:
this is a boolean

Reply:
not sure what this was ported from but none of these platforms would exercise this code path

https://www.appveyor.com/docs/environment-variables/ `CI=true`
https://buildkite.com/docs/pipelines/configure/environment-variables `CI=true`
https://circleci.com/docs/reference/variables/ `CI=true`
https://docs.drone.io/pipeline/environment/reference/ci/ `CI=true`
https://docs.github.com/en/actions/reference/workflows-and-actions/variables `CI=true`
https://docs.gitlab.com/ci/variables/predefined_variables/ `CI=true`
https://docs.travis-ci.com/user/environment-variables/#default-environment-variables `CI=true`